### ISSUE TITLE
react/no-unknown-property: allow viewbox on symbol

### ISSUE THREAD

**Issue by pgrippi** [jsx-eslint/eslint-plugin-react#3426]
Similar to https://github.com/jsx-eslint/eslint-plugin-react/issues/3416 but for `symbol` instead of `marker`

using version `7.31.7`

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox

I have a component that looks like:

```jsx
<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em">
  <symbol viewBox="0 0 24 24" id="..." xmlns="http://www.w3.org/2000/svg">
    <path
      fill="currentColor"
      d="..."
    />
  </symbol>
</svg>
```

And it's throwing an error:

```
error  Invalid property 'viewBox' found on tag 'symbol', but it is only allowed on: svg  react/no-unknown-property
```

it might be good to also add it for `pattern` and `view` as those tags also accept `viewBox` according to MDN but are not listed in the [allowed tags](https://github.com/jsx-eslint/eslint-plugin-react/blob/fd90b0e35a91d1e19849086a073a500fc66040c9/lib/rules/no-unknown-property.js#L66)

**Comment by ljharb** [comment]
This will be fixed by #3424
